pymagicc/__init__.py

```diff
@@ -303,28 +303,16 @@ def run(scenario, output_dir=None,
         write_scen_file(scenario,
                         os.path.join(model_run.run_dir, "SCENARIO.SCEN"))
 
-        all_cfgs = {}
-        years = {
-            "startyear": 1765,
-            "endyear": 2100,
-            "stepsperyear": 12
-        }
-
-        for k, v in kwargs.items():
-            if k in years.keys():
-                years[k] = v
-            else:
-                all_cfgs[k] = v
-
-        all_cfgs[get_param('emission_scenario_key')] = "SCENARIO.SCEN"
-        all_cfgs["rundate"] = _get_date_time_string()
-
-        # Write simple config file.
-        outpath = os.path.join(model_run.run_dir, "MAGTUNE_SIMPLE.CFG")
-        f90nml.write({"nml_allcfgs": all_cfgs}, outpath, force=True)
-        # Write years config.
-        outpath_years = os.path.join(model_run.run_dir, "MAGCFG_NMLYEARS.CFG")
-        f90nml.write({"nml_years": years}, outpath_years, force=True)
+        year_cfg = {}
+        if 'startyear' in kwargs:
```

Comment:
It would be nice to change this to the more pythonic `start_year`, but that is a slight api change

Reply:
I'd agree, but I think we should stick with the MAGICC names …